nubis/bin/example-ldap2consul.conf

```diff
@@ -0,0 +1,13 @@
+[ldap]
+server = ldap.domain.com
+binduser = somebinduser
+bindpass = agoodpass
+searchbase = dc=domain
+searchfilter = (cn=group)
```

Comment:
The problem here is that Mozilla's LDAP schema is unique. This happens at a lot of companies, actually, and it's why some of the most LDAP-compatible tools require parameters to be specified in LDAP syntax. Take a look at mana.mozilla.org's config some time to see what I mean - the only reason we've been able to integrate it so tightly to our LDAP is because it doesn't try to sugar coat LDAP settings.

While I also strive to make things as easy as possible for the end user, I have spent enough time on the other side of this problem (trying to get someone's LDAP-speaking tool/app to work with my org's LDAP infra) to know that the solution you propose makes things *harder*, not easier.